Mezzurite.React/src/with-mezzurite.hoc.js

```diff
@@ -101,17 +102,18 @@ const withMezzurite = (WrappedComponent) => {
                     if (entry.isIntersecting){
                         window.mezzurite.vltComponentLookup[that.fullName] = true;
                     }
-                    setTimeout(function(){
-                        const slow = PerformanceTimingService.calculateSlowestResource(el, that.fullName);
-                        if (slow === null){
-                            PerformanceTimingService.measure(that.fullName)
-                        }
-                        else{
-                            PerformanceTimingService.measure(that.fullName, slow)
-                        }
-                        el = null;
-                    },MezzuriteConstants.slowestResourceTimeout)
+                    // setTimeout(function(){

```

Comment:
I believe you removed this exact same function in angularjs-performance-component.ts, any reason for keeping it commented instead of removing it?

Reply:
Nope, I'll remove.